brokers/unified/broker.go

```diff
@@ -137,42 +142,84 @@ func (b *Broker) ProcessPlugins(metas []model.PluginMeta) error {
 	return nil
 }
 
+func ValidateMeta(meta model.PluginMeta) error {
```

Comment:
It's a little strange to me to make `ValidateMeta` an exported function but keep `validateMetas` internal. Is there a reason for this?

Reply:
An external program that would use brokers as a library could have the need to use this function on either 1 or several plugins.
Which is why the most useful public API call seemed to be the individual one. I didn't see any added value in exposing a function that simply loops on another basic API call.

Reply:
It's strange from a design point of view to have a more general internal function that calls an exported specialized function, and is contrary to what we do in general (e.g. in upstream Che).

Won't every app using this as a library have to wrap the calls to `ValidateMeta` in a loop? Most libraries would work with a slice of `PluginMeta`, as you can always pass in a slice of one element.

Reply:
The only project I know that uses the broker as a library is the Workspace CRD Controller :-) , and in this case, I manage each plugin separately because the overall plugin loop also includes other actions, such as creating the related k8s objects, etc... In fact it's simply the loop on devfile components. Sure I could refactor the code, but it's an example of the need to manage each plugin separately.
Additionally, to me proposing an atomic action in an API seems more general, or generic (in terms of use cases) than proposing to loop several times on the same action with a list of parameters, but here that's certainly a question of POV and wording ;-)